### PR TITLE
[Finishes #149338249] Added snapshots at Resource level

### PR DESCRIPTION
**Pull request by chalettu**
@miq-bot add_label enhancement
@miq-bot add_label fine/no
@miq-bot add_label  my services / snapshots
@miq-bot add_label ux/review

### CODE REVIEW

**Comment by chalettu** [comment]
Screenshot of the view action
snapshotsss1

**Comment by chalettu** [comment]
Create Snapshot Button 
create_snapshot

File: client/app/services/vm-details/vm-details.html [hunk 1]
```diff
@@ -37,9 +37,15 @@
...
                          config="item"
+                         ng-repeat="item in vm.snapshotListActions"
+                         menu-right="true">
+        </custom-dropdown>
+        <custom-dropdown class="custom-dropdown pull-left"
+                         config="item"
                          ng-repeat="item in vm.listActions"
                          menu-right="true">
         </custom-dropdown>
+
```
Comment (AllenBW) [inline client/app/services/vm-details/vm-details.html]: Is complaining about this unnecessary too pedantic? Unsure, gonna approve anyways, BUT ya should rebase this unnecessary space to OBLIVION 😁

**Comment by chriskacerguis** [comment]
For your review @serenamarie125 @Loicavenel

**Comment by Loicavenel** [comment]
looks good for me

**Comment by chriskacerguis** [comment]
Ping @serenamarie125 your thoughts / approval would be appreciated :)

**Comment by chriskacerguis** [comment]
@chalettu can you fix this...needs to be updated to use standard

**Comment by serenamarie125** [comment]
Upon create, is the modal completely in view @chalettu ?  It probably is, but can't tell

File: client/app/shared/action-button-group/action-button-group.component.js [hunk 2]
```diff
@@ -26,6 +27,7 @@ function ComponentController (sprintf) {
   vm.$onInit = function () {
     angular.extend(vm, {
       isDisabled: angular.isUndefined(vm.isDisabled) ? false : vm.isDisabled,
+      onSaveLabel: angular.isUndefined(vm.onSaveLabel) ? __('Save') : __(vm.onSaveLabel),
```
Comment (AllenBW) [inline client/app/shared/action-button-group/action-button-group.component.js]: > Don’t use dynamic content inside __()
http://manageiq.org/docs/guides/i18n

`s = __(sprintf("My name is %s.", me.name));`
The above should correctly be:
`s = sprintf(__("My name is %s."), me.name);`

**Comment by chalettu** [comment]
snapshot_text
@serenamarie125 , This dialog does center on the screen.  The capture I made didn't really demo that but it is centered.  I took the liberty of updating that button as requested.  Please let me know if you approve or have any more questions.

**Comment by serenamarie125** [comment]
@miq-bot  remove_labal ux/review
@miq-bot add_label ux/approved

**Comment by serenamarie125** [comment]
@miq-bot remove_label ux/review

**Comment by chriskacerguis** [comment]
Something must be wrong with coveralls / nsp...they aren't changing the status...going to force merge.

**Comment by chriskacerguis** [comment]
@AllenBW - @serenamarie125 has requested that we NOT remove the ux/required tag (regardless of approval or not) so they can easily find what they have been asked to review.

**Comment by AllenBW** [comment]
@chalettu duly noted!  noticed it was missing from the other approved pr, thought i would prune on this one, will stay away from next time 👍

**Pull request merged**